joeynmt/__main__.py

```diff
@@ -32,7 +35,8 @@ def main():
              output_path=args.output_path, save_attention=args.save_attention)
     elif args.mode == "translate":
         translate(cfg_file=args.config_path, ckpt=args.ckpt,
-                  output_path=args.output_path)
+                  output_path=args.output_path,
+                  n_best=args.nbest if args.nbest else 1)
```

Comment:
Instead of making the check here, use 1 as default for the nbest argument. In this way the user can easily find out about the behavior of the flag in one place.